src/App.js

```diff
@@ -1,6 +1,42 @@
-import React from 'react';
-import './App.css';
+import React, { Component } from "react";
```

Comment:
According to Airbnb eslint standards, we should use single quotes except when writing JSX.